### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by sydurand**
Add details to help to build the solution with Visual Studio 2017

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +36,21 @@ cmake -G "Visual Studio 16 2019" -A x64 -T v141_xp -DORC_BUILD_VCPKG=ON ..
...
 ```
 
+For Visual Studio 2017:
+```bash
+git clone https://github.com/dfir-orc/dfir-orc.git
+cd dfir-orc
+mkdir build-x86 build-x64
+
+cd build-x86
+cmake -G "Visual Studio 15 2017" -A Win32 -T v141_xp -DORC_BUILD_VCPKG=ON ..
```
Comment (jgautier-anssi) [inline README.md]: The correct command line should not include -A x64.

File: README.md [hunk 2]
```diff
@@ -35,6 +36,21 @@ cmake -G "Visual Studio 16 2019" -A x64 -T v141_xp -DORC_BUILD_VCPKG=ON ..
...
+cmake --build . --config MinSizeRel -- -maxcpucount
+
+cd ../build-x64
+cmake -G "Visual Studio 15 2017" -A x64 -T v141_xp -DORC_BUILD_VCPKG=ON ..
```
Comment (jgautier-anssi) [inline README.md]: The correct command line should be:
```
cmake -G "Visual Studio 15 2017 Win64" -T v141_xp -DORC_BUILD_VCPKG=ON ..
```

File: README.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ https://dfir-orc.github.io
 ## Build
 
 ### Requirements
-- Visual Studio >=2017 with this [configuration](.vsconfig) or alternatively use [vstools](docs/vstools/vstools.md)
+- Visual Studio >=2017 with this [configuration](.vsconfig) and "Desktop Developement C++" or alternatively use [vstools](docs/vstools/vstools.md)
```
Comment (jgautier-anssi) [inline README.md]: Why? Are you sure this requirement is not already selected via the .vsconfig ?

Comment (sydurand) [reply README.md]: With a fresh VS2017 install the import of .vsconfig file generate some warning and the "Desktop Developement C++" component are not checked.

File: README.md (hunk 1 above)
Comment (sydurand) [reply README.md]: Thank you for the correction

File: README.md (hunk 2 above)
Comment (sydurand) [reply README.md]: Thank you for the correction

**Pull request closed without merging**